src/AndersonAcceleration.f90

```diff
@@ -65,18 +65,18 @@ MODULE AndersonAccelerationTypes
     !> @copydetails AndersonAccelerationType::clear_AndersonAccelerationType
     PROCEDURE,PASS :: clear => clear_AndersonAccelerationType
     !> @copybrief AndersonAccelerationType::step_AndersonAccelerationType
-    !> @copydetails AndersonAccelerationType::step_AndersonAccelerationType
+    !> @copydetails NativesAndersonAccelerationType::step_AndersonAccelerationType
```

Comment:
this is different than the others?  why?

Reply:
Another hold over from having them split.